containers.tf

```diff
@@ -1,8 +1,8 @@
 
 resource "azurerm_storage_container" "container" {
-  for_each = var.containers
+  count = length(var.containers)
```

Comment:
don't use count that means that ordering changes will cause re-creation use for_each

you may need to add a `toset` here

Reply:
Couldn't use toset as doesn't support list of objects
Changed to use for_each

Reply:
Couldn't use toset as doesn't support list of objects
Changed to use for_each 

Tested with remote project in plan with containers